### PR TITLE
Update sort order for release note navigation

### LINKED ISSUE #381: Update sort order for release note nav
## Summary
Update migration so that the left nav for release notes (`src/nav/release-notes.yml`) should order release note files by `releaseDateTime` field (most recent release first).

We should automate how this nav is updated so content contributors don't have to edit this file each time they add a release note.

## Acceptance criteria
- [ ] As a end-user, I can see all release notes in order, most recent at the top, in the release notes nav.

## Related to
#349

### PR DESCRIPTION
**Pull request by caylahamann**
## Description 

Closes #381 

## Screenshots

Screen Shot 2020-12-18 at 6 51 42 PM

(ignore the duplication)
Screen Shot 2020-12-18 at 7 08 17 PM
sorted by release date!

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx [hunk 1]
```diff
@@ -1,38 +0,0 @@
----
```
Comment (jerelmiller) [inline src/content/docs/browser/new-relic-browser/getting-started/TEST-PAGE-Browser.mdx:1]: This page is currently being used for testing. Do you mind keeping this one around?

File: src/content/docs/browser/new-relic-browser/performance-quality/browser-monitoring-search-engine-optimization.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Browwwwser monstering and search engine optimization
+title: Browser monitoring and search engine optimization
```
Comment (jerelmiller) [inline src/content/docs/browser/new-relic-browser/performance-quality/browser-monitoring-search-engine-optimization.mdx:2]: This is currently being used for testing. Do you mind reverting this change?

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx [hunk 3]
```diff
@@ -220,33 +220,33 @@ Labels are optional tags which help to identify collection data. Some examples a
 
     Example `kafka-config.yml` config file for this environment:
 
-```
```
Comment (jerelmiller) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:223]: Unfortunately the site won't build with this change in place. I had to hand-edit this one for now. This should be fixed with the PR that recently got merged in MDX. For now, could you revert this change? I suspect this is what is causing the build to fail.

**Review by jerelmiller — CHANGES_REQUESTED** [review]
The code itself looks GREAT! Thanks so much for doing this.

The only reason I'm blocking this is because of the changes to the content itself. It might be easier just to revert the content completely. Would you mind doing that (should be able to do that with `git checkout develop -- src/content`)? I'd be happy to approve otherwise!

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx (hunk 3 above)
Comment (caylahamann) [reply src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:223]: yes I'll revert those changes!

**Review by jerelmiller — APPROVED** [review]
